src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts

```diff
@@ -199,22 +199,16 @@ export default class VideoPriorityBasedPolicy implements VideoDownlinkBandwidthP
     this.prevDownlinkStats = this.downlinkStats;
     this.downlinkStats = new LinkMediaStats();
     const metricReport = clientMetricReport.getObservableMetrics();
-    // Currently we have different stats between different browsers.
-    // availableReceiveBandwidth is the old legacy Google stat
+
     // availableIncomingBitrate is the standard stat but is not available in Firefox yet so only Safari for now.
-    if (!isNaN(metricReport.availableIncomingBitrate)) {
-      this.downlinkStats.bandwidthEstimateKbps = metricReport.availableIncomingBitrate / 1000;
-    } else {
-      this.downlinkStats.bandwidthEstimateKbps = metricReport.availableReceiveBandwidth / 1000;
-    }
+    this.downlinkStats.bandwidthEstimateKbps = metricReport.availableIncomingBitrate / 1000;
```

Comment:
What will it return for FF?

Reply:
sorry i meant would it be NaN or something similar

Reply:
it returns NaN in Firefox